### PR TITLE
Enable custom bias classes for umbrella integration

### PR DESCRIPTION
**Pull request by InnocentBug**

### CODE REVIEW

File: pysages/methods/harmonic_bias.py [hunk 1]
```diff
@@ -59,15 +59,13 @@ def __init__(self, cvs, kspring, center, **kwargs):
...
         """
-        super().__init__(cvs, **kwargs)
+        super().__init__(cvs, center, **kwargs)
         self.cv_dimension = len(cvs)
         self.kspring = kspring
-        self.center = center
 
     def __getstate__(self):
-        state, kwargs = default_getstate(self)
+        state, kwargs = super().__getstate__()
```
Comment (InnocentBug) [inline pysages/methods/harmonic_bias.py:67]: In general, shouldn't we implement `default_getstate` as the implementation of the general parent class?
Here of `SamplingMethod` and call `super().__getstate__()` in the inheriting classes.
It should work, that the parent class finds the correct `__special_args__` of the inherited classes and argument structure of the constructors, right?

Comment (pabloferz) [reply pysages/methods/harmonic_bias.py:67]: Yes. That should work as well. I believe I thought that we might want to do special pickling for CVs or other classes as well and that we could reuse the same code structure and base functions. But if there's reason to believe that won't be the case, we can replace this.

Comment (InnocentBug) [reply pysages/methods/harmonic_bias.py:67]: Ok, I can take care of this on Monday

**Comment by InnocentBug** [comment]
Locally, I cannot replicate the failiong MPI test.
I have to look in more detail later.

**Review by pabloferz — APPROVED** [review]

**Pull request merged**